### DISCUSSION TITLE
How to create a new user?

### DISCUSSION THREAD

**Discussion by gramian** [ArcadeData/arcadedb#543] [Q&A] [answered]
Dear Community,

how does one create a new user? OrientDB has a SQL command `CREATE USER <user> IDENTIFIED BY <password>`, this seems not to work here. I also did not find a syntax for the console, a button in the studio, or a command-line setting for the server.

Thanks

**Comment by lvca** [comment]
With ArcadeDB users can only be server users. There is no such concept as database users. This simplify a lot of things in comparison to OrientDB, especially from the run-time performance.

I realized the API to manage users/groups are not documented.

Below a piece of code that create and drop a user.

```java
ServerSecurity security = server.getSecurity();
security.createUser(new JSONObject().put("name", "elon").put("password", security.encodePassword("musk"))
        .put("databases", new JSONObject().put("test-database", new JSONArray(new String[] { "reader" }))));

security.dropUser("elon");
```

You can find more examples in `ServerProfilingIT` test class.

I'm going to create a new issue in the docs to create a chapter relative to user manipulation.

**Reply by lvca** [reply]
https://github.com/ArcadeData/arcadedb/issues/544

**Reply by gramian** [reply]
Thank you for the explanation, this makes sense; I guess I was confused by the description of the setting `server.defaultDatabases`, including a potential username and password which I interpreted as a database user.

To follow up: Is this (server-)user/group management API available from the console/studio/server (in the future?), so without "writing code"?

**Comment by gramian** [comment, marked as answer]
From version 22.12.1 on, a new user can be created via the [console](https://docs.arcadedb.com/#Console) or the [HTTP API](https://docs.arcadedb.com/#HTTP-CreateUser).
